src/utils.ts

```diff
@@ -0,0 +1,40 @@
+import { NextFunction, Request, Response } from "express";
+import fs from 'fs/promises'
+import Joi from "joi";
+
+export const getUserIdFromHeaders = (req: Request): number | null => {
+    const userId = req.headers['x-user-id'];
+
+    if (userId) {
+        return +userId;
+    }
+
+    return null;
+}
+
+export const readFromFile = async <T>(filePath: string): Promise<T> => {
+    return await fs.readFile(filePath).then(data => {
+        return JSON.parse(data.toString()) as T
+    });
+}
```

Comment:
I would rewrite it like this:

```suggestion
export const readFromFile = async <T>(filePath: string): Promise<T> => {
    const data = await fs.readFile(filePath);
    return JSON.parse(data.toString());
};
```

Reply:
done